python/bin/start.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -e
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Initialize functions and Constants
+BIN_DIR="$(dirname "$BASH_SOURCE")"
+PROJECT_ROOT_DIR=${BIN_DIR}/..
+
+PACKAGE_EGG_FILE=dist/dataproc_templates-0.0.1-py3.8.egg
```

Comment:
I am not very string on this one, but ...
- On one side since this name includes python version number, it will break for users with slightly different versions (e.g. Python 3.9)

- On the other side, since we specified Python 3.8 in requirements should we worry too much about supporting other versions?

I understand this is the default name when creating the Python egg, but name could easily be modified after creation.